### ISSUE TITLE
Add mouse support for search results and also make everything integration little bit simple like wox does

### ISSUE THREAD

**Issue by Manaswint** [oliverschwendener/ueli#588]

**Comment by oliverschwendener** [comment]
These are two different requests. As there is already a feature request for mouse interaction (#393) I'll remove the mouse interaction part from this issue.

**Comment by Manaswint** [comment]
Is update in development

**Comment by freedsny** [comment]
Yes please support this app, and it is very good although there is still room for improvement

**Comment by b-mq** [comment]
As a workaround you can build a custom search in UELI + Everything by activating the **URL protocol** in Everything and building a custom everything search in UELI via the **WebSearch plugin**, which starts with _es:_
_Everything URL Protocol_
![everything_url](https://user-images.githubusercontent.com/3727554/160345671-129be261-1fd1-4858-90f6-3aa796c001ff.png)
_Ueli Web Search Plugin_
![ueli_url](https://user-images.githubusercontent.com/3727554/160346264-e1d3c963-fb6b-4188-9d70-295c77573d98.png)
